src/Integration.Vsix/BoundSolutionAnalyzer.cs

```diff
@@ -94,6 +89,17 @@ private void OnSolutionBuilding()
             }
         }
 
+        private static string[] GetFiles(string rootDirectory, string folder, string pattern)
+        {
+            string[] matches = null;
```

Comment:
[minor][personal taste] I'd prefer to return `null` directly when directory doesn't exist and return `GetFiles` otherwise.